app/controllers/patients_controller.rb

```diff
@@ -314,6 +314,16 @@ def update_status
     patient = current_user.get_patient(params.permit(:id)[:id])
     redirect_to(root_url) && return if patient.nil?
 
+    # update LDE for patient and group members only in the exposure workflow with continuous exposure on (separate from updating monitoring status)
+    if params.permit(:apply_to_group_cm_exp_only)[:apply_to_group_cm_exp_only]
+      ([patient] + (current_user.get_patient(patient.responder_id)&.dependents&.where(continuous_exposure: true, isolation: false) || [])).uniq.each do |member|
```

Comment:
This line is so long and confusing. Let's simplify it. I propose creating a new method on patient (in the same style as `active_dependents` and `dependents_exclude_self` - so put it below those), that looks like:

```
def household
  [responder] + responder.dependents
end
```

and then this line can become

`patient.household.where(continuous_exposure: true, isolation: false).each do |member|`

I'm writing code in this github text field, so obviously untested...

Reply:
this actually got changed because we just want the dependents not the patient.  take a look at the changes and if you still feel it needs to change lmk